content/issues/2023-05-28-mariadb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Known Issue: MariaDB Check and Repair'
+date: '2023-05-28 12:00:00Z'
+severity: notice
+resolved: false
+affected:
+  - 'Known Issues'
+  - 'mariadb'
+section: 'issue'
+---
+There is a known issue with MariaDB when updating the container image and a database upgrade is required on user databases that are not in a healthy state. This can prevent MariaDB from starting.
+
+MariaDB version updates (ex: `10.6.x` to `10.11.x`) can happen when updating the container image. Upgrading user databases is done manually by running a command.
+
+When the container initializes, if `MYSQL_ROOT_PASSWORD` is set an upgrade check will run. If an upgrade is required the log will indicate the need to run:
```

Comment:
This needs a note about making sure anything using the database is stopped/disconnected before running the upgrade or check commands.

Reply:
Resolved in 5966df5b9120f16eeee8ba1379e7ec691c1a6197 also applied to https://github.com/linuxserver/docker-mariadb/pull/124 in https://github.com/linuxserver/docker-mariadb/pull/124/commits/bef18099cfbf718aa66d78cce629e1f4ffc9ce8c